.circleci/config.yml

```
@@ -16,3 +22,8 @@ jobs:
       - run: make testacc
     environment:
       GOPROXY: "https://proxy.golang.org"
+workflows:
+  version: 2
+  pipeline:
+    jobs:
+      - lint
```

The build job is not included in the pipeline and therefore not executed now.

```suggestion
      - lint
      - build
```

---

The build fails as the credentials are not configured in the pipeline

---

Ah, that's because #83 broke the pipeline for branches that aren't up to date. I'll merge `develop` back to this branch to fix the build.